gammapy/visualization/utils.py

```diff
@@ -20,6 +23,69 @@
 }
 
 
+def plot_rgb(map_, energy_edges=None, ax=None, **kwargs):
+    """
+    Plot RGB image on matplotlib WCS axes.
+
+    Based on the `~astropy.visualization.make_lupton_rgb` function.
+
+    Parameters
+    ----------
+    map_ : `~gammapy.maps.WcsNDMap`
+        WCS map. If energy edges are not provided, the map should contain exactly 3 energy bins.
+    energy_edges : `~astropy.units.Quantity`, float
+        Energy edges (an iterable of length 4) defining exactly 3 energy bins.
+    ax : `~astropy.visualization.wcsaxes.WCSAxes`, optional
+        WCS axis object to plot on.
+    **kwargs : dict
+        Keyword arguments passed to `~astropy.visualization.make_lupton_rgb`.
+
+    Returns
+    -------
+    ax : `~astropy.visualization.wcsaxes.WCSAxes`
+        WCS axis object
+
+    Examples
+    --------
+    >>> from gammapy.visualization.utils import plot_rgb
+    >>> from gammapy.maps import Map
+    >>> import astropy.units as u
+    >>> map_ = Map.read("$GAMMAPY_DATA/cta-1dc-gc/cta-1dc-gc.fits.gz")
+    >>> kwargs = {"stretch": 0.5, "Q": 1, "minimum": 0.15}
+    >>> plot_rgb(map_.smooth(0.08*u.deg), [0.1, 0.2, 0.5, 10] * u.TeV, **kwargs)
+    """
+    axis = map_.geom.axes["energy"]
+    if not energy_edges:
+        if len(axis.center) != 3:
+            raise ValueError("Exactly 3 energy bins are needed to plot an RGB image")
+        else:
+            energy_edges = axis.edges
+
+    if len(energy_edges) != 4:
+        raise ValueError("Exactly 3 energy bins are needed to plot an RGB image")
+
+    axis_rgb = MapAxis.from_energy_edges(
+        energy_edges.value, unit=energy_edges.unit, name="energy", interp=axis.interp
```

Comment:
```suggestion
        energy_edges.value, unit=energy_edges.unit, name=axis.name, interp=axis.interp
```
In case the input map is in true energy